ui/src/variables/utils/getWindowVars.ts

```diff
@@ -63,7 +64,15 @@ export const getWindowPeriod = (
       files: [ast, buildVarsOption(variables)],
     }
 
-    const queryDuration = getMinDurationFromAST(substitutedAST)
+    const queryDuration = getMinDurationFromAST(substitutedAST) // in ms
+
+    const foundDuration = SELECTABLE_TIME_RANGES.find(
+      tr => tr.seconds * 1000 == queryDuration
```

Comment:
`getMinDurationFromAST` returns a number, i think this can be `===`

Reply:
thanks!